src/hotspot/share/opto/compile.cpp

```diff
@@ -1105,13 +1105,12 @@ void Compile::Init(bool aliasing) {
   probe_alias_cache(nullptr)->_index = AliasIdxTop;
 }
 
-//---------------------------init_start----------------------------------------
+#ifdef ASSERT
 // Install the StartNode on this compile object.
```

Comment:
I think this comment needs to be updated.

Reply:
I believe the following may sound good briefly?: "Verify that the current StartNode is valid."

Reply:
Sounds good to me.